src/components/tag/_tag.scss

```diff
@@ -4,10 +4,15 @@
 
 @include govuk-exports("govuk/component/tag") {
   .govuk-tag {
-    @include govuk-font($size: 16, $weight: bold, $line-height: 1.25);
+    @include govuk-font($size: 16, $weight: bold, $line-height: 1);
 
     display: inline-block;
-    padding: govuk-spacing(1) 8px 0;
+    padding-top: .3125em;
```

Comment:
Given we've got #845 open to try and remove our only current usage of `em` for padding, do we want to be introducing it again here, or should we be using pixels?

Also, can this be a single `padding` rule as the values are all the same?

Reply:
I'd prefer to use relative units since it means when the tag is resized the padding is in proportion. Will show screenshots, before and after PX.

The padding results are not all the same, but I'll see if I can reduce the repeitition (I personally do not like shorthand since it's like doing `function(many, arguments, order, isimportant)` vs `function({ order: 'not important' })` etc)

Reply:
I think other components like the button keep the same dimensions when the font-size reduces (because the line-height is constant) so I'd expect this component to work in the same way… especially as we want to keep the /5 vertical rhythm.

Totally failed to notice that padding-bottom was different so… that's fine 👍

Reply:
OK, good to know, thanks Ollie.